pages/profiles.js

```diff
@@ -34,27 +37,94 @@ const Button = styled(ButtonBase).attrs({
   }
 `;
 
-const Profiles = ({ regions }) => (
-  <HeaderLayout
-    title="Country Profiles"
-    subtitle="Track progress and trends on achieving the Sustainable Development Goals for all 193 UN Member States"
-  >
-    <Box my={4}>
-      <Grid container spacing={2}>
-        {regions.map((region) => (
-          <ResponsiveGridItem key={region.id} small={3} medium={6} large={8}>
-            <Link href={profileUrl({ profile: region })} passHref>
-              <Button>
-                <RegionAvatar region={region} size={100} />
-                <Typography variant="body1">{region.name}</Typography>
-              </Button>
-            </Link>
-          </ResponsiveGridItem>
-        ))}
-      </Grid>
-    </Box>
-  </HeaderLayout>
-);
+const Profiles = ({ regions }) => {
+  const timeoutRef = useRef(null);
+  const [filter, setFilter] = useState(null);
+
+  useEffect(() => {
+    return () => {
+      if (timeoutRef.current != null) clearTimeout(timeoutRef.current);
+    };
+  }, []);
+
+  const onChange = (e) => {
+    if (timeoutRef.current != null) clearTimeout(timeoutRef.current);
+
+    timeoutRef.current = setTimeout(() => {
+      setFilter(e.target.value.toLowerCase());
+    }, 250);
+  };
+
+  const getName = (region) => `${region.name}`;
+
+  const isVisible = (region) => {
+    if (filter == null || filter === "") return true;
+    return getName(region)
+      .toLowerCase()
+      .normalize("NFD")
+      .replace(/[\u0300-\u036f]/g, "")
+      .includes(filter.normalize("NFD").replace(/[\u0300-\u036f]/g, ""));
+  };
+
+  return (
+    <HeaderLayout
+      title="Country Profiles"
+      subtitle="Track progress and trends on achieving the Sustainable Development Goals for all 193 UN Member States"
+    >
+      <Box my={4} style={{ flexGrow: 1 }}>
+        <Grid container spacing={2} alignItems="flex-end">
+          <Grid item style={{ width: "100%" }}>
```

Comment:
```suggestion
          <Grid item xs={12} sm={6} md={4} lg={3} >
```

Not sure if these are the right choices, but maybe you can try different ones and see?

It's a grid with 12 columns, so 12 = 100%. 6 = 50%. 3 = 25%. etc...

![image](https://user-images.githubusercontent.com/55237550/120210939-a527cd80-c230-11eb-8b84-5c1bfd5f0942.png)

Source: https://material-ui.com/customization/breakpoints/#breakpoints